nixos/modules/services/x11/desktop-managers/gnome.nix

```diff
@@ -215,7 +221,9 @@ in
       services.gnome.core-shell.enable = true;
       services.gnome.core-utilities.enable = mkDefault true;
 
-      services.displayManager.sessionPackages = [ pkgs.gnome-session.sessions ];
+      services.displayManager.sessionPackages = [
+        (pkgs.gnome-session.override { x11Support = !cfg.waylandOnly; }).sessions
```

Comment:
Lower, we use `gnome-session` for portal configs. Same in the `gdm` module. I wonder if we should use the overridden packages there as well.

Probably not necessary, since we only take the `sessions` output here.